src/components/Chip/Chip.tsx

```diff
@@ -22,25 +23,22 @@ const Chip: React.FC<Props> = ({ text, isSelected, shouldShake, onToggle }) => {
       <Text variant="chip" color={isSelected ? 'white' : 'blueDark'}>
         {text}
       </Text>
+
+      {isSelected && <Close />}
     </StyledChip>
   );
 };
 
-const shakeAnimation = keyframes`
-  10%, 90% { transform: translate3d(-1px, 0, 0); }
-  20%, 80% { transform: translate3d(2px, 0, 0); }
-  30%, 50%, 70% { transform: translate3d(-4px, 0, 0); }
-  40%, 60% { transform: translate3d(4px, 0, 0); }
-`;
-
 const StyledChip = styled.button<{ isSelected: boolean; shouldShake: boolean }>`
+  align-items: center;
   appearance: none;
   border: none;
   border-radius: 1.75rem;
   cursor: pointer;
+  display: flex;
   flex: 0 0 auto;
+  gap: 0.5rem;
   height: 2.75rem;
-  margin: 0.5rem;
```

Comment:
I removed the margin from the component. It's easier to use the component if there is no default-margin. The margin should come from outside. (I think so)

This also applies when we are using our Text-component in our application. The line-heights and font-sizes are so drastically different that there is quite a lot of margin. 
I would like to do one refactor PR where we remove the unnecessary margins, and then fix the style things from outside. It is difficult to work with compoennts when there is a HUGE default-margin =P or a line-height. 
I would prefer default to be 0 and then add if needed.

Reply:
I agree! This would be a good improvement.